src/main/java/levi9/UserService/controller/UserController.java

```diff
@@ -24,12 +25,19 @@ public UserController(UserService userService) {
         return ResponseEntity.ok(userService.register(newUser));
     }
 
-
-    @GetMapping(value = "/{username}")
+    //get user by its username
+    @GetMapping(value = "/getUser/{username}")
     public ResponseEntity<UserDto> getUserByUsername(@PathVariable String username){
         return new ResponseEntity<>(userService.loadUserByUsername(username), HttpStatus.OK);
     }
 
+    //update user
+    @CrossOrigin
+    @PutMapping("/updateUser/{username}")
```

Comment:
same as above, change to /user/{username}

Reply:
solved